ansible/roles/set-repositories/tasks/rhn-repos.yml

```diff
@@ -18,7 +18,27 @@
     pool_ids: "{{ rhsm_pool_ids }}"
     auto_attach: false
 
-- name: "Run 'subscription-manager to disable/enable repos"
+# TODO: should take pool ids from a var
```

Comment:
Look at this TODO

Reply:
We need to compare this file with upstream: https://github.com/redhat-cop/agnosticd/blame/development/ansible/roles/set-repositories/tasks/rhn-repos.yml